loadtests/loadtest/__init__.py

```diff
@@ -141,6 +146,20 @@ def read_further(self):
         }
         self._patch(self.random_url, data)
 
+    def batch_read_further(self):
+        data = {}
+        for i in range(25):
+            request = {
+                "path": self.random_url if i % 2 == 0 else self.random_url_2,
```

Comment:
So it's doing 25 PATCHes on the same two resources, changing the position randomly?

I find this weird :)